modules/search/instant-search/components/search-result.jsx

```diff
@@ -0,0 +1,42 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { sprintf, _n } from '@wordpress/i18n';
+import { h, Component } from 'preact';
+import strip from 'strip';
+
+class SearchResult extends Component {
+	render() {
+		return (
+			<div className="jetpack-instant-search__result">
+				<a
+					href={ `//${ this.props.result.fields[ 'permalink.url.raw' ] }` }
+					target="_blank"
+					rel="noopener noreferrer"
```

Comment:
I am not sure I understand why we need the links to open in a new tab here. Since we are on the same site, could we open them as is?

Reply:
👍 I'll make note of this in the master issue.